contracts/modules/cover/Cover.sol

```diff
@@ -390,7 +391,8 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
         start.toUint32(),
         period.toUint32(),
         productType.gracePeriod,
-        0 // global rewards ratio
+        0, // global rewards ratio
+        globalCapacityRatio
```

Comment:
Technically we won't burn any staking pool so the amount here is irelevant, but since we're (hypothetically) divide by it, can you set this to `1`, please?